### PR TITLE
Add CC (Número de Cartão de Cidadão, Portuguese Identity number)

### PR DESCRIPTION
**Pull request by davidmgvaz**
This adds CC (Número de Cartão de Cidadão, Portuguese Identity number)

### CODE REVIEW

File: stdnum/pt/cc.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301 USA
+
+"""CC (Número de Cartão de Cidadão, Portuguese Identity number).
+
+The Portuguese Identity Number is alfa numeric with the following format:
```
Comment (DimitriPapadopoulos) [inline stdnum/pt/cc.py:23]: alphanumeric

File: stdnum/pt/cc.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        value_sum += value
+
+    if value_sum % 10 != 0:
+        raise InvalidChecksum()
+
+    return number
+
+
+def is_valid(number):
+    """Check if the number is a valid VAT number."""
```
Comment (DimitriPapadopoulos) [inline stdnum/pt/cc.py:96]: ... a valid cartao de cidadao number

File: stdnum/pt/cc.py (hunk 1 above)
Comment (davidmgvaz) [reply stdnum/pt/cc.py]: Correct in last coomit

File: stdnum/pt/cc.py (hunk 2 above)
Comment (davidmgvaz) [reply stdnum/pt/cc.py]: Corrected in last commit

**Comment by davidmgvaz** [comment]
Hi @DimitriPapadopoulos,

I have already corrected the two typos you found and also split the calck_check_digit to an autonomous function, as I saw that was the standard practice

**Comment by arthurdejong** [comment]
Hi @davidmgvaz 

Thanks for the contribution. Could you provide some test numbers that can be used to confirm the implementation? For most number formats we have a number of tests with valid numbers to be able to validate the correctness of the calculations.

**Comment by arthurdejong** [comment]
Also a part of the number is a Número de Identificação Civil which has its own check digit. Do you have any idea (or pointers) how to validate that number?

**Comment by magamig** [comment]
@arthurdejong any particular reason to not update the README?

**Comment by arthurdejong** [comment]
The README and documentation are updated as part of the release process with [update/numlist.py](https://github.com/arthurdejong/python-stdnum/blob/master/update/numlist.py).

**Pull request closed without merging**